OxfordBodleian/Juel-Jensen/BDLaethf29.xml

```diff
@@ -715,34 +718,54 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                         </additions>
 
                         <bindingDesc>
-                            <binding contemporary="false" xml:id="binding">
-                                <decoNote xml:id="b1">Two<material key="wood">wooden</material>
-                                    boards covered with tooled reddish brown <material
-                                        key="leather"/>. A niche for a mirror is carved into the
-                                    inner side of the front board (the mirror is lost, but traces of
-                                    adhesive are still visible). The inner side of the back board is
-                                    decorated with a silk inlay (coloured gold, blue, and
-                                    red).</decoNote>
-                                <decoNote xml:id="b2" type="bindingMaterial">The textile inlays are
-                                    either cotton or silk (see <ref target="#e3"/>). <material
-                                        key="wood"/>
-                                    <material key="leather"/>
-                                    <material key="textile"/>
+                            <binding contemporary="true" xml:id="binding">
+                                <decoNote xml:id="b1" type="bindingMaterial">
+                                  The book is bound in <material key="wood">wooden</material>
+                                  boards covered with blind-tooled <material key="leather"/>.
+                                  The lower board is decorated with a <material key="textile"/> inlay.
```

Comment:
The keyword textileInlay exists now and could be used here, I think

Reply:
From now on I will use it!